parse.go

```diff
@@ -788,6 +807,22 @@ func isFlag(s string) bool {
 	return strings.HasPrefix(s, "-") && strings.TrimLeft(s, "-") != ""
 }
 
+func emptyLongAndShort(kv []string) bool {
+	var noShort, noLong bool
+	for _, key := range kv {
+		if key == "-" {
+			noShort = true
+		}
+
+		if key == "--" {
+			noLong = true
+		}
+	}
+
+	return noShort && noLong
+
+}
+
```

Comment:
seems like the need for this function could be avoided as per the comments above

Reply:
:heavy_check_mark: